x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_initializer.tsx

```diff
@@ -82,24 +118,36 @@ export const AnomalySwimlaneInitializer: FC<AnomalySwimlaneInitializerProps> = (
       (swimlaneType === SWIMLANE_TYPE.VIEW_BY && !!viewBySwimlaneFieldName));
 
   const resultInput = {
+    jobIds,
     panelTitle,
     swimlaneType,
     ...(viewBySwimlaneFieldName ? { viewBy: viewBySwimlaneFieldName } : {}),
   };
 
   return (
-    <EuiModal initialFocus="[name=panelTitle]" onClose={onCancel}>
-      <EuiModalHeader>
-        <EuiModalHeaderTitle>
-          <FormattedMessage
-            id="xpack.ml.swimlaneEmbeddable.setupModal.title"
-            defaultMessage="Anomaly swim lane configuration"
-          />
-        </EuiModalHeaderTitle>
-      </EuiModalHeader>
+    <>
+      <EuiFlyoutHeader>
```

Comment:
The flyout is not being hidden if you navigate away from Dashboard.

Reply:
Fixed in https://github.com/elastic/kibana/pull/182586/commits/e4240b7096e44d7de6a143b1071b3bd6e6a94a4a